src/_locales/en_US/messages.json

```diff
@@ -423,6 +423,10 @@
             }
         }
     },
+    "popup_info_firstparty_protections": {
+        "message": "Protection from <a href='https://www.eff.org/deeplinks/2018/05/privacy-badger-rolls-out-new-ways-fight-facebook-tracking'>link tracking</a> is enforced on this site",
```

Comment:
https://www.eff.org/deeplinks/2018/05/privacy-badger-rolls-out-new-ways-fight-facebook-tracking is specific to Facebook. Maybe link to https://privacybadger.org/#What-about-tracking-by-the-sites-I-actively-visit%2c-like-NYTimes.com-or-Facebook.com instead?